lib/sportradar/api/nfl/play.rb

```diff
@@ -19,6 +20,10 @@ def initialize(data)
         @sequence = data["sequence"]
         @start_situation = Sportradar::Api::Nfl::Situation.new data["start_situation"] if data["start_situation"]
         @statistics = OpenStruct.new data["statistics"] if data["statistics"] # TODO Implement statistics?
```

Comment:
These seem to be the different stats coming out of all those play types:
```
["ast_sack",
 "ast_tackle",
 "att_yards",
 "attempt",
 "category",
 "complete",
 "confirmed",
 "down",
 "endzone",
 "faircatch",
 "firstdown",
 "forced",
 "forced_fumble",
 "fumble",
 "goaltogo",
 "gross_yards",
 "inside_20",
 "int_yards",
 "interception",
 "lost",
 "missed",
 "nullified",
 "opp_rec",
 "opp_rec_yards",
 "out_of_bounds",
 "own_rec",
 "own_rec_yards",
 "pass_defended",
 "penalty",
 "player",
 "primary",
 "qb_hit",
 "reception",
 "return",
 "sack",
 "sack_yards",
 "tackle",
 "target",
 "team",
 "tlost",
 "tlost_yards",
 "touchback",
 "touchdown",
 "yards",
 "yards_after_catch"]
```

So there are about 45 different things across 13 categories; OpenStruct may continue to be ok for everything but those which we'd normally map to an object like `player` and `team`